pkg/controller/backup_deletion_controller.go

```diff
@@ -449,3 +478,20 @@ func (c *backupDeletionController) patchBackup(backup *v1.Backup, mutate func(*v
 
 	return backup, nil
 }
+
+func getObjectStoreForLocation(location *v1.BackupStorageLocation, manager plugin.Manager) (cloudprovider.ObjectStore, error) {
```

Comment:
Not sure if it needs to be addressed and if so if it should be in this PR: we use the alias `api` in other places, and `v1` here. Example: https://github.com/heptio/ark/blob/60c6624d60bb23c494aeca8d16c25073d4a7717a/pkg/controller/backup_controller.go#L475

Do we want to make it all the same?

Reply:
I am surprised GH is not saying there's a conflict since Nola's PR has been merged with this same function..

Reply:
> I am surprised GH is not saying there's a conflict since Nola's PR has been merged with this same function..

Probably because it's a different file. I expect the conflict will be caught at build time.